src/themes/teams/componentStyles.ts

```diff
@@ -52,4 +52,7 @@ export { default as Status } from './components/Status/statusStyles'
 
 export { default as Text } from './components/Text/textStyles'
 
+export { default as Tree } from './components/Tree/treeStyles'
+export { default as TreeListItem } from './components/Tree/treeListItemStyles'
+export { default as TreeTitle } from './components/Tree/treeTitleStyles'
 export { default as Transition } from './components/Transition/transitionStyles'
```

Comment:
nit: would suggest to put empty line after these three, to aid visual separation of components from different domains